public/index.html

```diff
@@ -16,7 +16,7 @@
   -->
     <meta
       http-equiv="Content-Security-Policy"
-      content="script-src 'self'; style-src 'self' 'unsafe-inline'; font-src http: data:"
+      content="script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; font-src http: data:"
```

Comment:
Why do you want this `unsafe-eval`?

Reply:
It's needed for the `survey` plugin, though I'm not sure why